src/libaudqt/prefs-plugin.cc

```diff
@@ -110,6 +110,10 @@ EXPORT void plugin_prefs (PluginHandle * ph)
             p->cleanup ();
 
         cw->root = nullptr;
+        // make sure the main preferences window is still in front when we exit
+        // when we close (won't be the case on Mac).
+        // Assumes that we were always called from that dialog!
+        prefswin_show ();
```

Comment:
What is the problem you're trying to fix here?

Reply:
This seems like it's working around a Qt and/or window manager bug.  Please talk to Qt and/or Apple to see if they are aware of the problem, and if this is really the right way to fix it.